content/source/docs/enterprise/sentinel/manage-policies.html.md

```diff
@@ -60,7 +60,7 @@ Sentinel policies themselves are defined in individual files (one per policy) in
 
 ## Managing Policy Sets
 
-To manage policy sets, go to the [organization settings][] and navigate to the "Policy Sets" page. Policy sets enforced on all workspaces are marked with a "GLOBAL" tag in this list; other policy sets show how many workspaces they are enforced on.
+To manage policy sets, go to the organization settings page (click "Settings" in the top navigation bar) and click the "Policy Sets" link on the left side of the page. Policy sets enforced on all workspaces are marked with a "GLOBAL" tag in this list; other policy sets show how many workspaces they are enforced on.
```

Comment:
Maybe have a small image here with the index page so its easy to visualize and you can cut some of the words?

To manage policy sets, go to Policy Sets in the organization Settings page. 

(image)

Policy sets enforced on all workspaces....

Reply:
I thought about this but didn't do it because we have this exact same thing in the getting started guide, so the lines are getting blurred a little. But since you're also thinking this, probably makes sense to just add it in here as well to be thorough.